recipes/salabim/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "salabim" %}
+{% set version = "24.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/salabim-{{ version }}.tar.gz
+  sha256: f76a91496eebcf8c02ca70fc9380a7bffc652156589c82d1726c4f11ad45e19a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+  run_constrained:
+    - greenlet
+    - numpy
+    - opencv
+    - pandas
+    - pillow
+    # salabim docs say this stricth pin is needed, but worried that this will
+    # be problematic in environment solves
+    - pyglet  # ==1.5.27
+    - pyopengl
+    - pysimplegui
+    - python-dateutil
+    - scipy
+    # no feedstock, but may be added in future
+    # - pywavefront
+
+test:
+  imports:
+    - salabim
+  commands:
+    - pip check
+  requires:
+    - pip
+
+outputs:
+
+  - name: salabim-animation
+    requirements:
+      build:
+        - pip
+        - python >=3.6
+      run:
+        - python >=3.6
+        - salabim
+        # all of these are optional dependencies but are needed for the
+        # animations
+        - numpy
+        - opencv
+        - pillow
+        - pyopengl
```

Comment:
Add a simple test for this pacakge (even a import salabim is fine) so that the run environment is created at least.

Reply:
Done